src/spatio_temporal_voxel_grid.cpp

```diff
@@ -214,8 +216,11 @@ void SpatioTemporalVoxelGrid::PopulateCostmapAndPointcloud(const \
 
   if (_pub_voxels)
   {
-    _pc->push_back(pcl::PointXYZ(pose_world[0], pose_world[1], \
-                                 pose_world[2]));
+    geometry_msgs::Point32 point;
+    point.x=pose_world[0];
```

Comment:
Add some spaces here just to be consistent

Reply:
Added